recipes/python-symengine/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
```

Comment:
This would require this PR ( https://github.com/conda/conda-build/pull/1002 ), which I don't believe shows up in a release yet.